docs/source/using/addons.adoc

```diff
@@ -161,6 +161,21 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+Enabled add-ons automatically get executed during link:../command-ref/minishift_start{outfilesuffix}[`minishift start`].
+However, you can also explicitly apply an add-on with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
+It works with both enabled and disabled add-ons.
```

Comment:
Opposed to the implicit add-on executions during `minishift start` which only applies enabled add-ons, `minishift addon apply` allows to apply enabled as well as disabled add-ons. You can also apply multiple add-ons with a single command by specifying multiple add-on names separate by space.